src/javascript/JContent/actions/zipUnzip/zipAction.jsx

```diff
@@ -33,7 +34,7 @@ export const ZipActionComponent = withNotifications()(({path, paths, render: Ren
         return (Loading && <Loading {...others}/>) || false;
     }
 
-    let isVisible = res.checksResult;
+    let isVisible = compareVersions(window.contextJsParameters.dxVersion, '8.1.3.0') < 0 && res.checksResult;
```

Comment:
The goal is to hide the zip action on behalf of "download as zip" in case of version >= 8.1.3.0 
the download as zip action is not available before 8.1.3.0